docs/standard/garbagecollection/fundamentals.md

```diff
@@ -151,7 +151,7 @@ Before a garbage collection starts, all managed threads are suspended except for
 
 The following illustration shows a thread that triggers a garbage collection and causes the other threads to be suspended.
 
-! [When a thread triggers a garbage collection](../images/IC393001.png)
+! [When a thread triggers a garbage collection](../media/IC393001.png)
```

Comment:
As I said, it's inside media, inside a folder with the topic name. So it would be ./media/fundamentals/.